CONTRIBUTING.md

```diff
@@ -17,26 +17,39 @@
     ```
 
 4. Have a look around to get a feel for the app.
-5. Run the tests. We only take pull requests with passing tests, and it's great to confirm that you have a clean slate.
+
+5. (Optional) Note that to be able to use the page as an admin, you must first give yourself admin privileges.
+  Make sure you have started your app in docker (if using locally, skip the docker-related parts in these instructions) and signed up as a user on your locally running app.
+  * First get your id:
+    * Run `docker ps` to get the id of your running postgres container id.
+    * Run `docker exec -it <your-id-here> psql -U postgres planner_development` where planner_development is the database name.
+    * Query `SELECT * from members WHERE name = '<Your first name>';` (remember the ; in the end of the line, it's very important!)
+    * Exit with `exit` twice.
+  * Then access rails console:
+    * Run `./bin/drails console`
+    * Then in the console run: `Member.find(<your-id>).add_role(:admin)`
```

Comment:
Thanks for your comments! I tried this with `"Member.where(name: 'Taru').add_role(:admin)"` but only got error:
```
/usr/local/bundle/gems/activerecord-4.2.11.1/lib/active_record/relation/delegation.rb:136:in `method_missing': D, [2019-10-20T15:44:22.443021 #1] DEBUG -- :   Member Load (1.1ms)  SELECT "members".* FROM "members" WHERE "members"."name" = $1  [["name", "Taru"]]
undefined method `add_role' for #<Member::ActiveRecord_Relation:0x00000000066e9b18> (NoMethodError)
```
But the other one `Member.last.add_role(:admin)` worked like a charm! I will add `dadmin`-script as you suggested, because I too think that's a much nicer way for solving this. I will also add your instructions for non-docker users to contributing-readme. 👍

Reply:
re: Where Oops.

### Where solution

`where` returns, sort of in the loosest way, an array of Members and an array won't know what an `add_role` method is so you got the "undefined method" exception. A fix is to get the actual member from the sort-of array: 

`Member.where(name: '<you first name'>).first.add_role(:admin)` 

Assuming your name is unique.

### find_by solution

`find_by` returns an actual Member and not an array of members. Assuming your the first person named 'Taru' *thinks* which I reckon there's a decent chance. 

`Member.find_by(name: '<your first name>').add_role(:admin)`



## References
[find_by](https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by)
[where](https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-where)

Reply:
Cool, thanks! I edited instructions and `dadmin` script to use `find_by`.